### ISSUE TITLE
thumbnails - compound object search results

### ISSUE THREAD

**Issue by brnco** [FutureDays/mfa#1]
With Islandora Default Thumbnails disabled, the current search results page looks like:
![mfa-search1](https://user-images.githubusercontent.com/15820937/31909520-4072f4ea-b7ef-11e7-9246-300c7fffba56.png)

Which is ugly, specifically the blank boxes on the left-hand side. Those are supposed to be thumbnail previews. In admin/islandora/solution_pack_config/compound_object we have checked the box for "Generate a thumbnail for compound objects from their first child"

What that means is, for objects ingested without children, no default thumbnail is generated (I think). 

This behavior would explain another error as well:
![mfa-search2](https://user-images.githubusercontent.com/15820937/31911211-c2e678c0-b7f4-11e7-9767-8db8c5903ae6.png)

For that, I just enabled the Islandora Default Thumbnails module.

I tried tracking down the error through this post: https://groups.google.com/forum/#!searchin/islandora/default$20thumbnail%7Csort:date/islandora/_vNLLy8-Ens/aUyGEbBrAwAJ

and changed the corresponding line in query_processor.inc. that gave me the following error
![mfa-search3](https://user-images.githubusercontent.com/15820937/31911316-1fc4a0d0-b7f5-11e7-9725-18d8c2491991.png)

Basically, there's an array that expects to have [object_label] as a key, it doesn't exist I think because these objects don't have children. IDK
